docs/howto/local-checkouts.md

```diff
@@ -39,10 +40,72 @@ storage, resulting in cost savings.
 
 <iframe width="420" height="315" src="https://www.youtube.com/embed/afgQnmesLZM"></iframe>
 
-## **lakectl local**: The way to work with lakeFS data locally  
+## **lakeFS Mount**: Efficiently expose lakeFS Data as a local directory
+
+⚠️ lakeFS Mount is currently in preview. There is no installation required, please [contact us](https://info.lakefs.io/thanks-lakefs-mounts) to get access.
+{: .note }
+
+#### Prerequisites:
+
+- A working lakeFS Server running either [lakeFS Enterprise](../enterprise) or [lakeFS Cloud](../cloud)
+- You’ve installed the [`lakectl`](../reference/cli.html) command line utility: this is the official lakeFS command line interface, on top of which lakeFS Mount is built.
```

Comment:
Why is lakectl a requirement?

Reply:
because it sets up `~/.lakectl.yaml` properly. While technically you don't really need the binary itself, once you have it set up, you'll be able to use everest as well.

Reply:
I find this to be confusing. Because even if you download `lakectl` you still don't have `~/.lakectl.yaml` file which means you'll get an error when running `everest`.
Can you be explicit about the need for `lakectl.yaml` configuration file? it can even be empty one but this is the actual pre-requisite.